crates/wasmcloud-host/src/host_controller/hc_actor.rs

```diff
@@ -341,6 +342,11 @@ impl Handler<StopActor> for HostController {
                     interest: WasmCloudEntity::Actor(pk.to_string()),
                 });
                 act.actors.remove(&pk);
+                let _ = cp.do_send(PublishEvent {
+                    event: ControlEvent::ActorStopped {
+                        actor: pk.to_string(),
```

Comment:
per clippy: this can just be `pk`, doesn't need to be cloned here since this is the last thing that will use pk.